### PR TITLE
CHANGELOG.md: Add 0.6.0 entries

### PR DESCRIPTION
**Pull request by dmajda**

### CODE REVIEW

**Comment by dmajda** [comment]
@demmer

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,20 +2,56 @@
...
+
+Released 2016-03-09
+
+### Major Changes
+
+- Enabled runaway program detection. [[#473](https://github.com/juttle/juttle/issues/473)]
+
+### Minor Changes
+
+- The `.` operator can now be used to access object properties (not just module exports). [[#497](https://github.com/juttle/juttle/pull/497)]
```
Comment (demmer) [inline CHANGELOG.md]: I think it'd be useful to clarify what situations this extends the language to support nested points support for, i.e. rvalues for `put`, assignments in functions, etc.

Depending on how useful this is, I'd consider this a major change perhaps.

**Comment by demmer** [comment]
looks good otherwise.

**Comment by dmajda** [comment]
@demmer I clarified the `.` operator entry and moved it to Major Changes (I was split on this myself). I also extended the `sigma` changes to cover stdlib and added entries for work done since yesterday.

@bkutil Could you check the above so that I can release? (It’s the last 3 commits.)

**Comment by dmajda** [comment]
(I’ll squash before merge.)

**Comment by bkutil** [comment]
The extra changes look ok. +1

**Pull request merged**